src/main/java/seedu/duke/logic/commands/RouteMapCommand.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.logic.commands;
+
+import seedu.duke.BusRoute;
+import seedu.duke.exceptions.CustomException;
+import seedu.duke.ui.Ui;
+
+public class RouteMapCommand extends Command {
+
+    @Override
+    public void executeCommand() {
+        try {
+            Ui.printRouteSelectionMessage();
+            String userBusRouteSelection = Ui.getCommand().trim().toLowerCase();
+            System.out.println(BusRoute.selectBusRoute(userBusRouteSelection));
```

Comment:
Once you change the code according to the new classes handling bus data, you would still have a similar function that returns the bus route. I suggest you shift the printing to Ui class. See #26 for an example.